### PR TITLE
[candi] added FAQ about using Nvidia GPU with containerd

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
## Description

1. Added FAQ about setting up containerd with Nvidia GPU support.
2. Fixed work of simple-bridge in Yandex cloud with older kernels.
## Why do we need it, and what problem does it solve?

Simple-bridge CNI failed to work in Yandex Cloud on Debian9/Centos 7 due to error executing of command:
```
$ ip r get 0.0.0.1
RTNETLINK answers: Invalid argument
```
To fix this problem we use more real address, like `ip r get 1.1.1.1`.
## Changelog entries

```changes
section: candi
type: feature
summary: added FAQ about using Nvidia GPU with containerd
impact: low
```

### CODE REVIEW

**Review by yalosev — DISMISSED** [review]
Wall of text :)

File: modules/040-node-manager/docs/FAQ.md [hunk 1]
```diff
@@ -396,3 +397,756 @@ When changing the CRI in the cluster, additional steps are required for the mast
...
+metadata:
+  name: gpu
+spec:
+  chaos:
+    mode: Disabled
+  cri:
+    type: NotManaged
+  disruptions:
+    approvalMode: Automatic
+  nodeType: CloudPermanent
```
Comment (konstantin-axenov) [inline modules/040-node-manager/docs/FAQ.md:418]:
```suggestion
  nodeType: CloudStatic
```
CloudPermanent — nodes from ProviderClusterConfiguration will be created via dhctl;

**Review by konstantin-axenov — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]
I've committed some fixes

**Comment by z9r5** [comment]
> @z9r5 Please, check the docs.

I've committed some fixes

**Comment by z9r5** [comment]
> @z9r5 Please, check the docs.

I've committed some fixes

**Pull request merged**